pom.xml

```diff
@@ -0,0 +1,346 @@
+    <prerequisites>
+        <maven>3.0.3</maven>
+    </prerequisites>
+
+    <scm>
+        <url>https://github.com/scalecube/scalecube</url>
+        <connection>scm:git:git@github.com:scalecube/scalecube-benchmarks.git</connection>
+        <developerConnection>scm:git:git@github.com:scalecube/scalecube-benchmarks.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>scalecube.io</id>
+            <name>The ScaleCube Project Contributors</name>
+            <email>info@scalecube.io</email>
+            <organization>The ScaleCube Project</organization>
+            <organizationUrl>http://scalecube.io/</organizationUrl>
+        </developer>
+    </developers>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <modules>
+        <module>benchmarks</module>
+        <module>example</module>
+    </modules>
+
+    <properties>
+        <argLine>-Xms1024m -Xmx2048m</argLine>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <uberjar.name>scalecube-benchmarks</uberjar.name>
+        <scalecube-services-api.version>2.0.26</scalecube-services-api.version>
+        <reactor-core.version>3.1.7.RELEASE</reactor-core.version>
+        <jmh.version>1.21</jmh.version>
+        <metrics.version>3.1.2</metrics.version>
+
+        <junit.jupiter.version>5.2.0</junit.jupiter.version>
+        <hamcrest-all.version>1.3</hamcrest-all.version>
+
+        <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
+        <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+        <maven-javadoc-plugin.version>2.10.4</maven-javadoc-plugin.version>
+        <maven-shade-plugin.version>3.1.0</maven-shade-plugin.version>
+        <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+
+            <dependency>
+                <groupId>io.projectreactor</groupId>
+                <artifactId>reactor-test</artifactId>
+                <version>${reactor-core.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.dropwizard.metrics</groupId>
+                <artifactId>metrics-core</artifactId>
+                <version>${metrics.version}</version>
+            </dependency>
+
+            <!-- Other dependencies -->
+            <dependency>
+                <groupId>org.openjdk.jmh</groupId>
+                <artifactId>jmh-core</artifactId>
+                <version>${jmh.version}</version>
```

Comment:
Add `<optional>true</optional>` and  `<scope>provided</scope>`

Reply:
done